src/app/features/stacks-transaction-request/hooks/use-stacks-transaction-summary.ts

```diff
@@ -72,13 +73,21 @@ export function useStacksTransactionSummary(token: CryptoCurrencies) {
     };
   }
 
+  function getSip10MemoDisplayText(tx: StacksTransaction): string {
```

Comment:
Rather than casting again, we can pass just `ContractCallPayload` here

Reply:
Apologies, I didn't notice as I was trying to fix it quickly.